content/en/docs/reference/helper-libraries/dra-driver-kubelet-plugin.md

```diff
@@ -0,0 +1,40 @@
+---
+title: DRA Resource Driver controller helper library
+description: Handles most of functionality common for all DRA resource drivers' kubelet-plugins.
+weight: 20
+---
+
+<!-- overview -->
+
+This library implements some of the functionality that the
+[Dynamic Resource Allocation](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/) (DRA)
+resource-drivers' kubelet-plugins need to implement, that is common for all of
```

Comment:
```suggestion
resource drivers' kubelet plugins need to implement, that is common for all of
```

I can't think of a reason why hyphens should be used here? We don't do that elsewhere.

Reply:
I agree, I'd omit the hyphens.

Reply:
Erm. I was quite consistent (I think) in having a hyphen in kubelet plugin, so I'll mass-remove it from the Docs. I've no idea why I thought we put it there.

Reply:
Fixed.